src/test/java/com/bobocode/hoverla/bring/context/BeanDefinitionMapperTest.java

```diff
@@ -0,0 +1,31 @@
+package com.bobocode.hoverla.bring.context;
+
+import com.bobocode.hoverla.bring.helper.BeanDefinitionAssert;
+import com.bobocode.hoverla.bring.testsubject.config.TestBeanConfig;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Method;
+
+class BeanDefinitionMapperTest {
+
+    private BeanDefinitionMapper mapper;
+
+    @BeforeEach
```

Comment:
I would recommend removing this `setUp()` method and initialize `mapper` direcly on line `13`

Reply:
I don't see nothing wrong in using setup method, even if it's really that simple. In my opinion with setup test class looks more accurate

Reply:
But OK, I'll change it

Reply:
well, I left this remark only because we have single test method here. Thus in this case it looks like a couple of unnecessary  lines of code. But I suppose in the future there will be more tests, and perhaps this init method will get back :)) Holywar....

Reply:
### HOLYWAR,  HOLYWAR, HOLYWAR 🔥 🔥 🔥